tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -1026,17 +1059,19 @@ def test_continuous_reads_during_pod_restart(self):
             thread1.start()
         LOGGER.info("Step 5: Successfully started READs and verified DI on the written data in "
                     "background")
-        LOGGER.info("Step 6: Starting pod again by creating deployment using K8s command")
+        LOGGER.info("Step 6: Starting pod again and checking cluster status")
```

Comment:
We have to pass event_set_clr list to s3 bench IOs for background IOs. Or those changes we will do in separate PR?

Reply:
As we are expecting pass for all IO, I don't think we need to set it.